ecommerce/api.py

```diff
@@ -325,7 +325,9 @@ def complete_successful_order(order):
     run = order.get_bootcamp_run()
     application = order.application
     if application.is_paid_in_full:
-        BootcampRunEnrollment.objects.get_or_create(user=order.user, bootcamp_run=run)
+        BootcampRunEnrollment.objects.get_or_create(
+            user=order.user, bootcamp_run=run, active=True
```

Comment:
I think this would break the unique constraint if there was already an enrollment with active=False. We probably need to just get_or_create it, then set `active=True` and `enrollment.save()`

Reply:
How about:
```
BootcampRunEnrollment.objects.update_or_create(
            user=order.user,
            bootcamp_run=run,
            defaults={"active": True, "change_status": None},
        )
```